src/builder.rs

```diff
@@ -456,6 +456,9 @@ impl ActionInfo {
 /// This is returned by [`Builder::build`].
 pub type UnauthorizedBundle<V, D> = Bundle<InProgress<Unproven, Unauthorized>, V, D>;
 
+/// Type alias for an authorized bundle that has proofs and signatures.
+pub type AuthorizedBundle<V, D> = Bundle<Authorized, V, D>;
```

Comment:
This adds an abstruction layer with little benefit. The lenght is the same and it actually decreases readability. The user need to check what is `AuthorizedBundle` but gets nothing except more mental load.

Unlike `UnautorizedBundle`, where we needed it shorter becuase types became very long.

Please revert to using the original type.

Reply:
Done